app/route_rangers_api/views.py

```diff
@@ -31,15 +33,29 @@ def dashboard(request, city: str):
     # get commute
 
     # get paths
-    routes = TransitRoute.objects.filter(city="CHI").values(
-        "geo_representation", "route_name", "color"
+    routes = TransitRoute.objects.filter(city=CITY_CONTEXT[city]["DB_Name"])  # .values(
+    # MultiLineString needs to be serialized into a GeoJson object for Leaflet to
+    # work with it.
+    # to serialize into GeoJson, need to get out entire Django model object, not just
+    # the .values("geo_representation", "route_name", "color")
+    # with .values() you get "AttributeError: 'dict' has no component 'meta'"
```

Comment:
great note, do you notice slowness from loading all of these routes? If so maybe this is a candidate for caching since this data will generally not be changing

Reply:
I tested this out and it took about a minute to load the chicago page the first time, we may need to look into ways to cache, or auto-load on initialization, or only load some layers at once to reduce how much is being pulled in at a time.

Reply:
This took a minute or two for me to load too. I'm also wondering if splitting this into two functions may be better the pins need to be dynamically loaded and the routes don't.

Reply:
Yep, it's taking maybe 30-45 seconds to load and this is probably why. Agree that refactoring to speed it up should be its own ticket